app/case-management/case-management.js

```diff
@@ -13,9 +13,9 @@ const caseDataDao = {
         try {
             let queryParts = {};
 
-            var columns = "identifiers,DATE(encounter_datetime) AS last_follow_up_date,DATE(effective_rtc) AS rtc_date,extract(year from (from_days(datediff(now(),t1.birthdate)))) as age,  " +
-                "case_manager_name AS case_manager,person_name AS patient_name,gender,t1.vl_1 AS last_vl, t1.vl_1_date, TIMESTAMPDIFF(DAY,DATE(encounter_datetime),curdate()) AS days_since_follow_up,  " +
-                "uuid as patient_uuid, null as phone_rtc_date, null as case_manager_uuid, (CASE WHEN TIMESTAMPDIFF(DAY,DATE(rtc_date),curdate()) > 0 THEN 1 ELSE 0 END) as missed_appointment, " + getDueForVl() + "AS patients_due_for_vl ";
+            var columns = "identifiers,DATE_FORMAT(encounter_datetime, '%Y-%m-%d') AS last_follow_up_date,DATE_FORMAT(effective_rtc, '%Y-%m-%d') AS rtc_date,extract(year from (from_days(datediff(now(),t1.birthdate)))) as age,  " +
+                "case_manager_name AS case_manager,person_name AS patient_name,gender,t1.vl_1 AS last_vl, DATE_FORMAT(t1.vl_1_date, '%Y-%m-%d') as vl_1_date, TIMESTAMPDIFF(DAY,DATE(encounter_datetime),curdate()) AS days_since_follow_up,  " +
+                "uuid as patient_uuid, null as phone_rtc_date, case_manager_user_id as case_manager_uuid, (CASE WHEN TIMESTAMPDIFF(DAY,DATE(rtc_date),curdate()) > 0 THEN 1 ELSE 0 END) as missed_appointment, " + getDueForVl() + "AS patients_due_for_vl ";
```

Comment:
Why are we renaming case_manager_user_id to case_manager_uuid?

Reply:
I did this to avoid changes in the client because they were using **case_manager_uuid** already.

Reply:
I think they need to change it at the client to use the right column, otherwise it will be very confusing when you are trying to maintain this 3-4 months down the line. Ask them to change maybe?

Reply:
Okay